src/test/java/uk/gov/companieshouse/filinghistory/consumer/mapper/pretransform/EmbeddedChildNodeMapperTest.java

```diff
@@ -8,56 +8,60 @@
 import com.fasterxml.jackson.databind.ObjectMapper;
 import com.fasterxml.jackson.databind.node.ObjectNode;
 import java.util.List;
+import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.extension.ExtendWith;
 import org.junit.jupiter.params.ParameterizedTest;
 import org.junit.jupiter.params.provider.CsvSource;
-import org.mockito.InjectMocks;
 import org.mockito.Mock;
 import org.mockito.junit.jupiter.MockitoExtension;
 import uk.gov.companieshouse.api.delta.ChildProperties;
 import uk.gov.companieshouse.api.delta.FilingHistory;
+import uk.gov.companieshouse.filinghistory.consumer.transformrules.TransformerTestingUtils;
 import uk.gov.companieshouse.filinghistory.consumer.transformrules.functions.FormatDate;
 
 @ExtendWith(MockitoExtension.class)
 class EmbeddedChildNodeMapperTest {
 
-    @InjectMocks
-    private EmbeddedChildNodeMapper embeddedChildNodeMapper;
+    private static final ObjectMapper objectMapper = TransformerTestingUtils.getMapper();
 
-    @Mock
-    private ObjectMapper objectMapper;
+    private EmbeddedChildNodeMapper embeddedChildNodeMapper;
     @Mock
     private FormatDate formatDate;
 
+    @BeforeEach
+    void setUp() {
+        embeddedChildNodeMapper = new EmbeddedChildNodeMapper(objectMapper, formatDate);
```

Comment:
Do we need this? The value can be assigned once in the declaration, assuming the mapper is stateless

Reply:
@cgregoryCH explained that this fixes a mock ordering issue